tinyms/serving/server/server.py

```diff
@@ -159,14 +168,24 @@ class Server:
     Args:
         host (str): Serving server host ip. Default: '127.0.0.1'.
         port (int): Serving server listen port. Default: 5000.
+        serving_path (str, optional): Set the read path of a service configuration
 
     Examples:
         >>> from tinyms.serving import Server
         >>>
         >>> server = Server()
     '''
 
-    def __init__(self, host='127.0.0.1', port=5000):
+    def __init__(self, host='127.0.0.1', port=5000, serving_path=None):
+        json_data = {}
+        json_data.update({
```

Comment:
Please add an assertion that if `serving_path` is None, then there is no need to dump the json_data.